recipe/meta.yaml

```
@@ -29,6 +38,10 @@ about:
   license_family: GPL
   license_file: LICENSE.txt
```

```suggestion
  license_file: LICENSE  # [not linux]
  license_file: LICENSE.txt  # [linux]
```
https://github.com/conda-forge/shellcheck-feedstock/pull/15/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR43-R44

---

Won't do for OSX